app/controllers/albums.js

```diff
@@ -13,3 +14,19 @@ exports.listAlbums = (req, res, next) => {
     res.status(200).send(json.data);
   });
 };
+
+exports.buyAlbum = (req, res, next) => {
+  const decoded = sessionsManager.decode(req.headers.authorization);
```

Comment:
this token decode is being duplicated, you are already doing this in the auth middleware. So you can save that info somewhere so here you can re-use it, for example you can store it inside the `req` object